packages/mendel-pipeline/src/client/generators.js

```diff
@@ -60,6 +68,12 @@ class MendelGenerators {
             this.perform(bundle, doneBundles);
         });
 
+        // If printEntryCount is set, print number of entries collected
+        // by each generator.
+        if (this.printEntryCount) {
```

Comment:
I think we deprecated that in favor of `require('debug')('verbose:mendel:....')`